cmd/fossa/cmd/test/test_test.go

```diff
@@ -0,0 +1,169 @@
+package test
```

Comment:
I would call this file `integration_test.go`.

Reply:
I'm not convinced that this package is a full integration test because it is not connecting with the app.fossa.io API. This makes me feel like this is more in line with an end to end test and I'm hesitant to call it an integration test.

Regardless the package does need to be changed, thank you for pointing this out.